### PR TITLE
Kitchen testing for Wazuh deployment with Puppet

### PR DESCRIPTION
**Pull request by rshad**
Hi all!

This `PR` implementes the changes requested in the issue  https://github.com/wazuh/wazuh-puppet/issues/124. 

In this `PR` we implemented `Kitchen` testing for Wazuh deployment with Puppet, implementing tests for `wazuh-manager` and `wazuh-agent` installation and agent registration, on different distribuitions such as Ubuntu and CentOS.

Kr,

Rshad

### CODE REVIEW

File: manifests/agent.pp [hunk 1]
```diff
@@ -216,7 +216,7 @@
       }else{
         fail('This ossec module has not been tested on your distribution')
       }
-    }'Debian', 'debian':{
+    }'Debian', 'debian', 'Ubuntu', 'ubuntu':{
```
Comment (jm404) [inline manifests/agent.pp]: Maybe this changes that affect `agent.pp` and `manager.pp` should be implemented in another issue in order to have better tracking of it,

File: manifests/manager.pp [hunk 2]
```diff
@@ -294,7 +294,7 @@
       }else{
         fail('This ossec module has not been tested on your distribution')
       }
-    }'Debian', 'debian':{
+    }'Debian', 'debian', 'Ubuntu', 'ubuntu':{
```
Comment (jm404) [inline manifests/manager.pp]: Also, this change should be on another issue and its corresponding PR for tracking purposes.

File: manifests/agent.pp (hunk 1 above)
Comment (rshad) [reply manifests/agent.pp]: Hi Jose!

Thank you for reviewing!

Here I created a new issue related to such change: https://github.com/wazuh/wazuh-puppet/issues/140

Kr,

Rshad

File: manifests/manager.pp (hunk 2 above)
Comment (rshad) [reply manifests/manager.pp]: Hi Jose!

Thank you for reviewing!

Here I created a new issue related to such change: https://github.com/wazuh/wazuh-puppet/issues/140

Kr,

Rshad

**Pull request merged**